pkgs/tools/misc/osinfo-db-tools/default.nix

```diff
@@ -1,18 +1,18 @@
 { stdenv, fetchurl, pkgconfig, intltool, glib, libxml2
-, libxslt, libarchive, bzip2, lzma, json-glib
+, libxslt, libarchive, bzip2, lzma, json-glib, libsoup
 }:
 
 stdenv.mkDerivation rec {
   pname = "osinfo-db-tools";
-  version = "1.5.0";
+  version = "1.6.0";
 
   src = fetchurl {
     url = "https://releases.pagure.org/libosinfo/${pname}-${version}.tar.gz";
-    sha256 = "1pihjwajmahldxi3isnq6wcsbwj0hsnq8z5kp3w4j615ygrn0cgl";
+    sha256 = "0x155d4hqz7mabgqvgydqjm9d8aabc78vr0v0pnsp9vkdlcv3mfh";
   };
 
   nativeBuildInputs = [ pkgconfig intltool ];
```

Comment:
Could you replace `intltool` with `gettext`? We forgot to do that when we upgraded to 1.4.0:

https://gitlab.com/libosinfo/libosinfo/blob/v1.4.0/NEWS#L9

Same for libosinfo.

Reply:
i had to add perl to the nativeBuildInputs as well because it requires the `pod2man` command